docs/tutorials/building_r.md

```diff
@@ -4,87 +4,86 @@
 ### Build Setup (Without Recommended Packages)
 **1) Environment Variables**
 
-- We have environment variables for setting the paths for building R and storing the source code.
-- The path ENV variable for R Build and R Source code are BUILDDIR and TOP_SRCDIR respectively.
+- We have environment variables defining paths to directories for building R and storing the source code.
+- `BUILDDIR` defines the build directory: /workspaces/r-dev-env/build/r-devel
+`TOP_SRCDIR` defines the source directory: /workspaces/r-dev-env/svn/r-devel
 - The environment variables are set in the codespace image and are available when the codespace starts.
 
-  ![image](https://github.com/r-devel/r-dev-env/assets/72031540/6b6cb661-cde3-4c69-845a-a6a281c18ec2)
-
+  ![alt text](../assets/rdev6.png) //newss
+  
 **2) svn checkout**
 
-- The svn checkout cmd lets us create working of a repository with specific tag/branch.
-- Command:
+- The svn checkout command lets us create a local copy of a specific tag/branch of a repository.
+- We can check out the latest version of the trunk (the main branch) of the R sources to $TOP_SRCDIR as follows:
    ```bash
-     svn checkout https://svn.r-project.org/R/trunk/ "$TOP_SRCDIR"
+     svn checkout https://svn.r-project.org/R/trunk/ $TOP_SRCDIR
    ```
 - Output : We get file structure something like this after checking out R source code from R svn repository.
 
-   ![image](https://github.com/r-devel/r-dev-env/assets/72031540/2bce81e5-5ac7-45ec-b925-9222b0919cd8)
+   ![alt text](../assets/rdev7.png)//newss
 
-**3) Change Directory to BUILDDIR**
+**3) Change to the build directory**
    
-- We need to change our directory to R build directory(BUILDDIR) to build and configure our R source code.
+- To keep the source directory clean, we change to a build directory to configure and build R.
 
-- First we will create a directory using env var BUILDDIR.
-- Command:
+- First create the directory specified by the BUILDDIR environment variable.
 ```bash
 mkdir -p $BUILDDIR
 ```
 
-- Then we can change directory from root to $BUILDDIR one.
-- Command:
+- Then we can change directory from root to the build directory.
 ```bash
 cd $BUILDDIR
 ```

```

Comment:
Need to add in step here for downloading the source of recommended packages, i.e. copy step 3 "Download the source code for the recommended packages" from the old "Build Setup (With Recommended Packages)" section. All the numbering for this section will need updating!!